website/docs/configurations/01-network-operator.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Network Operator
+---
+
+This guide details recommended configurations for Network Operator v25.1.0 to enable RDMA over InfiniBand, optimized for AKS environments with Mellanox NICs.
+
+:::tip
+This guide assumes a basic understanding of Network Operator and its role in Kubernetes clusters. Readers unfamiliar with the Network Operator are advised to review the official [Getting Started Guide](https://docs.nvidia.com/networking/display/kubernetes2501/getting-started-kubernetes.html) before proceeding. The concepts and recommended configurations presented here build on that foundation to enable RDMA over InfiniBand in AKS. This documentation is based on Network Operator v25.1.0.
+:::
+
+## Recommended Configuration
+
+This guide details recommended configurations for Network Operator v25.1.0 to enable RDMA over InfiniBand, optimized for AKS environments with Mellanox NICs.
+
+
+### Helm Values
+
+Network Operator is deployed using [Helm](https://helm.sh/), and the [default Helm values](https://github.com/Mellanox/network-operator/blob/v25.1.0/deployment/network-operator/values.yaml) are recommended unless specific customizations are required. These defaults include [Node Feature Discovery (NFD)](https://kubernetes-sigs.github.io/node-feature-discovery/stable/get-started/index.html), a critical dependency that labels nodes with hardware details (e.g., Mellanox NIC presence) for pod scheduling.
+
+```bash
+helm repo add nvidia https://helm.ngc.nvidia.com/nvidia
+helm repo update
+helm upgrade --install --create-namespace -n network-operator network-operator nvidia/network-operator
+```
+
+### NicClusterPolicy
+
+Post-installation, create a `NicClusterPolicy` Custom Resource (CR) to define the desired state of networking components, such as Mellanox driver version and which device plugins to deploy. Two configurations are provided below: SR-IOV Device Plugin for exclusive Network Interface Card (NIC) access and RDMA Shared Device Plugin for shared access.
+
+#### SR-IOV Device Plugin
+
+The SR-IOV Device Plugin assigns each InfiniBand-enabled NIC (e.g., Mellanox ConnectX-6) to a single pod as a Kubernetes resource (`rdma/ib`). The number of available resources matches the count of physical NICs on the node (e.g., 1 NIC = 1 resource), ideal for workloads requiring maximum performance and isolation.
+
+```yaml reference
+https://github.com/Azure/aks-rdma-infiniband/blob/main/configs/sriov-device-plugin/sriov.yaml
+```
+
```

Comment:
Before showing an example on how to deploy a sample application as a pod, how about we also tell them how to deploy this:
 
```suggestion
To deploy the above config run the following command:

kubectl apply -k https://github.com/Azure/aks-rdma-infiniband/configs/sriov-device-plugin
```